src/analysis/type_enclosing.ml

```diff
@@ -33,6 +33,26 @@ let from_nodes ~path =
     | Module_declaration_name {md_type = {mty_type = m}}
     | Module_type_declaration_name {mtd_type = Some {mty_type = m}} ->
       ret (Modtype (env, m))
+    | Class_field
+        { cf_desc =
+            Tcf_method (_, _, Tcfk_concrete (_, {exp_type = { desc = Tarrow (_, _, t, _) } }))
```

Comment:
Can you break this long line ?

Reply:
I did. Feel free to adjust further as the formatting is still a little weird.